eml/components/lectures/VideoPlayer.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { forwardRef, useEffect, useState } from 'react';
+
+import { View, StyleSheet, Dimensions } from 'react-native';
+import { Video } from 'expo-av';
+
+
+
+import tailwindConfig from '../../tailwind.config';
+
+const CustomExpoVideoPlayer = forwardRef(({ onStatusUpdate, videoUrl, isMuted = false, isPlaying = true }, ref) => {
+    const screenWidth = Dimensions.get('window').width;
+    const screenHeight = Dimensions.get('window').height;
+
+    const styles = StyleSheet.create({
```

Comment:
Again this should be nativewind

Reply:
for some reason naitvewind does not work with the expo video player, so we have to use nativewind

Reply:
The expo videoplayer does not work properly with nativewind, so we have to use the stylesheet